static/bidder-params/sa_lunamedia.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Sa_Lunamedia Adapter Params",
+    "description": "A schema which validates params accepted by the Sa_Lunamedia adapter",
+    "type": "object",
+    "properties": {
+      "key": {
+        "type": "string",
+        "description": "network or placement key"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["network", "publisher"]
+      }
+    },
+    "required": ["key"]
+  }
```

Comment:
@SyntaxNode there is no `openrtb_ext/imp_{bidder}.go` file and no `ImpExt{Bidder}` despite there are some bidder specific parameters are expected (key and type). Is it ok not to have this file?

Reply:
We have added ExtImpSaLunamedia. But we didn't use it in the adapter. Do we need to use it for validation? I thought that such a check is already happening at the prebid-server level ?

Reply:
Yes, validation happens on Prebid server level based on sa_lunamedia.json file. You don't need to explicitly validate it in adapter code.